next.config.js

```diff
@@ -3,7 +3,21 @@ module.exports = {
   webpack: (config) => {
     config.module.rules.push({
       test: /\.svg$/,
-      use: ['@svgr/webpack'],
+      use: [
+        {
+          loader: '@svgr/webpack',
+          options: {
+            throwIfNamespace: false,
+            svgoConfig: {
+              plugins: [
+                {
+                  mergePaths: false,
```

Comment:
Does this fix the multi shapes filled in at once bug?

Reply:
Some of them! I did a bunch of manual edits on the images too.